sotodlib/io/imprinter.py

```diff
@@ -148,6 +148,35 @@ class Imprinter:
     def __init__(self, im_config=None, db_args={}, logger=None):
         """Imprinter manages the book database.
 
+        Example configuration file::
+        
+          db_path: imprinter.db
+          
+          sources:
+            tel-tube1:
+              g3tsmurf: g3tsmurf config file
+              slots:
+                - stream_id1
+                - stream_id2
+                - stream_id3
+            tel-tube2:
+              g3tsmurf: g3tsmurf config file
+              slots:
+                - stream_id4
+                - stream_id5
+          
+          hk_sources:
+            daq-node:
+              g3tsmurf: g3tsmurf config file
+
+        The different tel-tubes will be bound into separate books containing 
+        only the stream_ids listed under slots. While the hk_sources take a 
+        g3tsmurf config key, that is just for symmetry, those files only need
+        to include a data_prefix and g3thk_db entry if only hk books are being 
+        created from that configuration. The g3tsmurf configurations are assumed
+        to be one per daq node, meaning this example could have the same config
+        file repeated several times.
+
```

Comment:
I assume the config file isn't documented elsewhere.  Questions to add answers for (?):

- These notes are helpful.  Maybe two paragraphs, first for "sources", then for hk_sources (pointing out the differences).
- What determines the number of slots in the slotmask (i.e. 111 vs 1111111)
- Are you supposed to put a None as a slot entry, to have a gap?
- The "tel-tube" and "daq-node" keys end up getting used for ... something.  In the code, what is that called?  Just "teltube"?  And that in turn becomes part of the book name.  (What I'm getting at is that "daq-node" sounds like the name of a computer when really it's just the label we're going to apply to all the data from some particular source.  Some clarification would be helpful.)